setup.py

```diff
@@ -190,7 +190,7 @@ def run_tests(self):
             'osc2midi = osc2midi.main:main [osc2midi]',
         ]
     },
-    tests_require=['pytest'],
+    tests_require=['pytest','mock'],
```

Comment:
A space after the comma, please.

I recommend checking Python files with `pydocstyle` as a habit.